tests/event_config.at

```diff
@@ -58,3 +58,130 @@ TS_MAIN
 }
 TS_RETURN_MAIN
 ]])
+
+## ------------------------ ##
+## get_options_with_err_msg ##
+## ------------------------ ##
+
+AT_TESTFUN([get_options_with_err_msg],
+[[
```

Comment:
Please join these two lines above, otherwise gcc errors/warnings (even runtime error messages) will be pointing to wrong line numbers.